modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2ExportingActivationKeys.testcase

```diff
@@ -0,0 +1,88 @@
+@component-name = "portal-solutions"
+definition {
+
+	property osgi.modules.includes = "osb-site-initializer-customer-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Customer Portal";
+
+	var testSiteName = "Customer Portal Name";
+	var testSiteURL = "customer-portal-name";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new Customer Portal Site") {
+			Site.openSitesAdmin();
+
+			TeamExtranet.addSiteTemplateCP(
+				siteName = "${testSiteName}",
+				siteTemplateName = "Liferay OSB Site Initializer Customer Portal");
+
+			ApplicationsMenu.gotoSite(key_site = "${testSiteName}");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "${testSiteName}");
+
+			CustomerPortal.customerPortalTearDown();
+		}
+	}
+
+	@description = "LPS-148063 - Verify if the option to 'Export all key details (csv)' is available when clicking Actions."
+	@priority = "4"
+	test ExportAllKeyOptionsIsPresent {
+		task ("When a user selects a project in Customer Portal") {
+			CustomerPortal.enterOnProject(title = "Test Account 01");
+
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Portal");
+		}
+
+		task ("Then the user should be able to assert the 'Export All Key Details (csv)' option in the Actions dropdown") {
+			CP2NavigationMenu.assertActionsOption(key_action = "Export All Key Details (csv)");
+		}
+	}
+
```

Comment:
@epark18 isn't this scenario already covered by SuccessMessageWhenKeyIsDownloaded? I know that we have different test cases for it in the Test Map, but do you think that only 1 test case could cover both scenarios?